dice.js

```diff
@@ -0,0 +1,13 @@
+const dice = function() {
+    firstDice = Math.floor(Math.random() * (6 - 1 + 1) ) + 1;
```

Comment:
you can use thisu will generate a random number from 1 up 6 

`let diceOne = Math.floor(Math.random()*6)+1;`